pkg/api/v20220904/openshiftinstallversions_example.go

```diff
@@ -0,0 +1,10 @@
+package v20220904
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+// ExampleInstallOpenShiftVersions returns an example
+// InstallOpenShiftVersions object i.e []string that the RP would return to an end-user.
+func ExampleInstallOpenShiftVersionsResponse() interface{} {
+	return &InstallOpenShiftVersions{"4.10.20"}
```

Comment:
are we returning a string, or an array? this example is just a string?

Reply:
It is []string. `InstallOpenShiftVersions` is of type []string.
https://github.com/anshulvermapatel/ARO-RP/blob/list-version-endpoint/pkg/api/v20220904/openshiftinstallversions.go#L6-L7